### PR TITLE
[READY]fixes certain things having a 0 cooldown, makes spells recharged by processing instead of spawn.

### PR DESCRIPTION
**Pull request by silicons**
gunhog was right i need to stop using the web editor

### CODE REVIEW

File: code/modules/spells/spell.dm [hunk 1]
```diff
@@ -254,14 +254,15 @@ GLOBAL_LIST_INIT(spells, typesof(/obj/effect/proc_holder/spell)) //needed for th
...
+/obj/effect/proc_holder/spell/proc/perform(list/targets, recharge = TRUE, mob/user = usr) //if recharge is started is important for the trigger spells
 	before_cast(targets)
 	invocation(user)
 	if(user && user.ckey)
 		user.log_message("<span class='danger'>cast the spell [name].</span>", INDIVIDUAL_ATTACK_LOG)
-	spawn(0)
-		if(charge_type == "recharge" && recharge)
-			start_recharge()
+	if(recharge)
+		spawn(0)
```
Comment (Jalleo) [inline code/modules/spells/spell.dm]: Is it better with sleep or spawn here or timer?

Comment (Cyberboss) [inline code/modules/spells/spell.dm]: STOP ADDING SPAWNS REEEE!!!

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/spells/spell.dm (hunk 1 above)
Comment (kevinz000) [inline code/modules/spells/spell.dm]: im not adding spawns there ALREADY WAS ONE!!

Comment (Cyberboss) [inline code/modules/spells/spell.dm]: FIX IT!

**Review by Cyberboss — CHANGES_REQUESTED** [review]
nvm

**Pull request merged**